sechub-sereco/src/test/java/com/mercedesbenz/sechub/sereco/WorkspaceTest.java

```diff
@@ -59,9 +73,36 @@ void do_import__no_importer() throws IOException {
         /* formatter:on */
 
         /* execute */
-        IOException exception = assertThrows(IOException.class, () -> workspace.doImport(importParameter));
+        IOException exception = assertThrows(IOException.class, () -> workspace.doImport(new SecHubConfiguration(), importParameter));
 
         /* test */
         assertEquals("Import failed, no importer was able to import product result: " + productId, exception.getMessage());
     }
+
+    @Test
+    void do_import_does_call_sensitive_data_masking_service() throws IOException {
+        /* prepare */
+        SecHubMessage info = new SecHubMessage(SecHubMessageType.INFO, "info");
+        String importId = "id1";
+        String importData = TestFileReader.loadTextFile("src/test/resources/sarif/sarif_2.1.0_owasp_zap.json");
+        String productId = "PDS_WEBSCAN";
+        List<SecHubMessage> messages = List.of(info);
+
+        /* formatter:off */
+        ImportParameter importParameter = ImportParameter.builder().importData(importData).importId(importId).importProductMessages(messages)
```

Comment:
You are using formatter:off here, so please indent
```suggestion
        ImportParameter importParameter =     
        
        ImportParameter.builder().
                  importData(importData).
                  importId(importId).
                  importProductMessages(messages).
                  productId(productId).
        build();
```
Or use no formatter off/on here.